superset/assets/src/addSlice/AddSliceContainer.jsx

```diff
@@ -79,15 +80,16 @@ export default class AddSliceContainer extends React.PureComponent {
       <div className="container">
         <Panel header={<h3>{t('Create a new chart')}</h3>}>
           <div>
-            <p>{t('Choose a datasource')}</p>
+            <p>
+              {t('Choose a')}{' '}<GlossaryWord word="datasource" />
```

Comment:
I don't think this will translate correctly. If a language doesn't have the same structure as English, then it might not make sense for "datasource" to come after "Choose a"

Also, the addition of a space isn't always correct (e.g. in Japanese or Chinese)

Reply:
Agreed, I wonder if we need a superset of markdown with some support for "Superset glossary" on top, something like: `t('Choose a {{{datasource}}}')`. I probably won't have time to get this deep anytime soon.

The context for this PR is I wanted to improve the "New Chart" form to allow people to create a new datasource inline (expanding to show a `<TableSelector />`), and thought we really needed to clarify what a datasource is in that context first, and got side-tracked into this glossary PR.

Reply:
In the future translation support for a "superset glossary" would be cool. Maybe for now it would make sense to apply the tooltip to the entire phrase? So that you don't need to worry about translation difficulties